packages/web-components/src/components/va-link-action/va-link-action.css

```diff
@@ -85,4 +85,84 @@
   vertical-align: top;
   max-width: calc(100% - 24px);
   white-space: pre-wrap;
+}
+
+:host(.host--primary-entry) {
+  width: 100%;
+}
+
+:host a.va-link--primary-entry {
+  position: relative;
+  display: inline-block;
+  background-color: var(--vads-color-primary);
+  color: var(--vads-color-white);
+  text-decoration: none;
+  padding: 15px 15px 15px 45px;
+  margin: 0 1.5rem 0 0;
+  clip-path: polygon(0 0, calc(100% - 15px) 0, 100% 50%, calc(100% - 15px) 100%, 0 100%);
+  transition: all 0.2s ease-in-out;
+  width: 95%;
+  line-height: 1.4;
+  min-height: 24px;
+  max-width: var(--vads-size-line-length-5);
+
+  @media (min-width: 768px) {
+    width: 75%;
+  }
+
+  & va-icon.link-icon {
+    position: absolute;
+    left: 12px;
+    top: 50%;
+    transform: translateY(-50%);
+    color: var(--vads-color-primary);
+    background-color: var(--vads-color-white);
+  }
+
+  & .link-text {
+    font-weight: bold;
+    display: inline-block;
+    vertical-align: middle;
+  }
+
+  /* Hover state */
+  &:hover:not(:focus) {
+    text-decoration: none;
+
+    & va-icon.link-icon {
+      background-color: var(--vads-color-action-focus-on-light);
+    }
+    & .link-text {
+      color: var(--vads-color-action-focus-on-light)
+    }
+  }
+
+  /* Focus & Active state */
```

Comment:
There's some similarity with this action link variation and the recently added [va-critical-action component](https://adds-va-link-action-primary-4001--65a6e2ed2314f7b8f98609d8.chromatic.com/?path=/docs/components-va-critical-action--docs). The focus states seem very different though.

![Screenshot 2025-04-11 at 9 05 33 AM](https://github.com/user-attachments/assets/ea687735-2621-4257-b3ec-7385680fb9ad)

![Screenshot 2025-04-11 at 9 07 10 AM](https://github.com/user-attachments/assets/60d4b3e2-e3e8-4f33-a2cd-def118cc1755)

Should we consider syncing these up?

Reply:
Yeah, this was definitely on my mind while creating this (my component was first to production though!) That lighter background for the critical action component is because it's already using the focus state yellow. The team explored changing it to another color but, we couldn't agree on something we liked. In short, we think this is maybe okay? We're going to find out when critical action gets tested.